cli/command/plugin/list.go

```diff
@@ -43,21 +42,19 @@ func runList(dockerCli *command.DockerCli, opts listOptions) error {
 		return err
 	}
 
-	w := tabwriter.NewWriter(dockerCli.Out(), 20, 1, 3, ' ', 0)
-	fmt.Fprintf(w, "ID \tNAME \tDESCRIPTION\tENABLED")
-	fmt.Fprintf(w, "\n")
-
-	for _, p := range plugins {
-		id := p.ID
-		desc := strings.Replace(p.Config.Description, "\n", " ", -1)
-		desc = strings.Replace(desc, "\r", " ", -1)
-		if !opts.noTrunc {
-			id = stringid.TruncateID(p.ID)
-			desc = stringutils.Ellipsis(desc, 45)
+	format := opts.format
+	if len(format) == 0 {
+		if len(dockerCli.ConfigFile().PluginsFormat) > 0 && !opts.quiet {
```

Comment:
Would be nice to add an integration test for config.json. Something like `TestPsDefaultFormatAndQuiet`